pkg/deploy/deployment_postgres.go

```diff
@@ -20,14 +20,14 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func NewPostgresDeployment(cr *orgv1.CheCluster, chePostgresPassword string) *appsv1.Deployment {
+func NewPostgresDeployment(cr *orgv1.CheCluster, chePostgresPassword string, isOpenshift bool) *appsv1.Deployment {
```

Comment:
I don't speak golang but will this break any existing implementations which don't have the bool param?

Reply:
I'm not aware that the `NewPostgresDeployment` is used in any other repo than the `che-operator` repo. Did I miss smething ?
Is this code used a a library in some external repo ?

If not, it shouldn't hurt since I updated the [only one call site](https://github.com/eclipse/che-operator/pull/30/files#diff-d9445e614f0d728fd140771b0fd29c50L311).

Afaict, this method is public only because it was defined in a distinct package.